_posts/2025-09-10_community-survey-findings.md

```diff
@@ -0,0 +1,178 @@
+10. Steam Publication
+
+First, we welcome all feedback and we have multiple ways for users to submit feedback to us. The best way to provide feedback it so
+get on GitHub and direclty file issues with us [VS Engine Bug](https://github.com/vegastrike/Vega-Strike-Engine-Source/issues),
+[VS:Upon The Coldest Sea Bug](https://github.com/vegastrike/Assets-Masters/issues), or engage with us in conversation on [Gitter.im](https://gitter.im/vegastrike/community).
+Alternative we do have some email addresses:
+
+- [Development](mailto:devel@lists.vega-strike.org)
+- [Security](mailto:security@lists.vega-strike.org)
+
+And of course there is the [VS Forums](https://forums.vega-strike.org/).
+
+While the team is most active on Gitter.im and GitHub, we do also pay attention to the other methods as well.
+
+### MMO Support
+
+The original developers had built in some multi-player support similar to that of Wing Commander Armada and other games from the 1990's where one could direct-connect
+with another user to play. However, we received feedback from those involved with the original dev team that it was not working and as our new dev team was just getting
+familiar with the code we made the decision to remove that support for the time being. We also had to initially drop support for Windows and Mac due to what our dev
+team could manage.
+
+Our first target was to get the game functional again, which we were able to do with the 0.7.0 and 0.8.0 releases. After that we wanted to re-introduce support
+for Windows and Mac, which was accomplished with the 0.9.0 release; though due to changes on the Mac platform we may have to drop support there temporarily again due
+to issues on the M-series processors. For 0.10.0 we are working on settings and other functionality and the team has developed enough expertise to begin making some
+more radical refactors as we work our way towards the 1.0 mark where we have a solid game that we can build from. There is still much to do.
+
+Now regarding multi-player support in any form, the current idea is to target that work for the 2.0 releases. And the best way to speed that up would be to
+join us on the development team or sponsor developers so we could have developers with more dedicated time to Vega Strike - at present all our devs are 100% volunteer
+and also have to be able to support themselves and their families via other jobs.
+
+### Walk the Planet
+
+From some conversations with members involved with the old dev team we are aware there were some thoughts around doing this kind of thing. However, we do not ourselves
+know yet how to extend the game for this. So presently we will need to keep this in the backlog for a future addition so we can focus on getting to that 1.0 release mark.
+
+### In-Game 3D Map of the VS Universe
+
+We are not sure where this presently falls. Certainly an in-game map would be a new feature and we do not yet have bandwidth to take on a new feature. However, we might
```

Comment:
We could possibly extend the 3d map of the system or just do a 2d map. It could be more of a graph than a map.